### PR TITLE
Added database functionality: 

### PR DESCRIPTION
**Pull request by Jonestj1**
creates database documents, updates database documents, query the database for specific documents, and retrieves all the documents in a database

### CODE REVIEW

File: metamds/db.py [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+    user : str, optional 
+        User name (default is None, meaning the database is public).
+    password : str, optional 
+        User password (default is None, meaning there is no password access to database).
+    collection : str, optional 
+        Database collection name for doc location (default is tasks).
+    **kwargs : keys of lists of strings (ie {"A": ["a"], "B": ["b", "bb"]} or A=["a"])
+        Fields and field values being queried in a simulation.
+    
+    Yields
```
Comment (ctk3b) [inline metamds/db.py]: Should be `Returns`

Same in the functions below

**Comment by ctk3b** [comment]
Thanks @Jonestj1! Could you come up with a set of unit tests? Maybe using a minimal (very small, single molecule) trajectory?

**Comment by Jonestj1** [comment]
yeah

On Thu, Jul 14, 2016 at 9:37 AM, Christoph Klein <notifications@github.com>
wrote:

> Thanks @Jonestj1 <https://github.com/Jonestj1>! Could you come up with a
> set of unit tests? Maybe using a minimal (very small, single molecule)
> trajectory?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/iModels/metamds/pull/4#issuecomment-232684351>, or mute
> the thread
> <https://github.com/notifications/unsubscribe/ALHQ8g4QWDLuUN4CF8UufFBk6XPAQx1jks5qVkmxgaJpZM4JLnVm>
> .
>



-- 
Trevor J Jones
Vanderbilt University
Chemical and Biomolecular Engineering 2017
email: Trevor.J.Jones@Vanderbilt.edu
cell: (513) 679-1457

**Comment by ctk3b** [comment]
Did you figure out how to create a temp database with py.test?

**Pull request merged**